source/databases-collections.txt

```diff
@@ -119,31 +119,39 @@ initial collection.
 
 To create a new database:
```

Comment:
[suggestion, minimalism]

I know this was pre-existing, but I don't think we need the lines "To create a new database:" and "To create a new collection:". They repeat the headings, and I think can be removed without impacting user understanding.